students/heb2016/session06/mailroom_unittest.py

```
@@ -0,0 +1,11 @@
+
+import unittest
+from mailroom.py import input_name
+
+def test_system(self):
+        raise SystemExit(1)
+        assert input_name('MS')
+ 
```

what are you trying to test here?

In any case, you don't generally want to raise an Exception in a test function.